topsail/_common.py

```diff
@@ -162,6 +162,23 @@ def _run(self):
         print(f"Using '{env['ANSIBLE_CACHE_PLUGIN_CONNECTION']}' to store ansible facts.")
         Path(env["ANSIBLE_CACHE_PLUGIN_CONNECTION"]).parent.mkdir(parents=True, exist_ok=True)
 
+        # We configure the roles path dynamically appending them to the defaults
+        top_dir_roles = Path(top_dir / "roles")
```

Comment:
```
top_dir = Path(__file__).resolve().parent.parent
```
`top_dir is already a Path, so:
```suggestion
        top_dir_roles = top_dir / "roles"
```